packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -156,26 +159,26 @@ const EDiscovery = SparkPlugin.extend({
 
   @waitForValue('@')
   /**
-   * Retrieves a list of the conversations relevant to a specified report, paged with a specified offset and size
-   * @param {UUID} reportId - Id of the report being requested
-   * @param {long} offset - the starting index of the results the caller wants
-   * @param {long} size - the number of conversations the caller wants
+   * Retrieves information for a specific conversation relevant to a specified report
+   * @param {UUID} reportId - Id of the report which contains the relevant content summary
+   * @param {UUID} spaceId = Id of the conversation being requested
    * @returns {Promise<ResponseEntity<ContentSummary>>} Http response containing the specified content summary
    */
-  getPagedContentSummary(reportId, offset, size) {
-    if (!reportId) {
-      throw Error('Invalid parameter(s)');
+  _getContentSummaryBySpaceId(reportId, spaceId) {
+    if (!reportId || !spaceId) {
+      throw Error('Undefined parameter(s)');
+    }
+
+    // If the content summary for this report has already been cached then it can be retrieved from there instead of making a network call to ediscovery
+    if (this.contentSummaryCache.has(reportId) && this.contentSummaryCache.get(reportId).has(spaceId)) {
+      return this.contentSummaryCache.get(reportId).get(spaceId);
```

Comment:
getPagedContentSummary is no longer required by the UI, getContentSummaryBySpaceId is used for retrieving specific information relating to a space